docs/utilities/tags.md

```diff
@@ -21,17 +21,45 @@ Using Tags In Code
 ------------------
 Block, Item, and Fluid tags are automatically sent from the server to any remote clients on login and reload. Function tags are not synced.
 
+Tags can be accessed in code through the `Tags.Items` and `Tags.Blocks` classes for all the Vanilla Minecraft tags, and through `ResourceLocation`s for ForgeTags.
+See the examples below for how to use each.
+
 `BlockTags.getCollection()` and `ItemTags.getCollection()` will retrieve the current `TagCollection`, from which you can retrieve a `Tag` object by its ID.
 With a `Tag` object in hand, membership can be tested with `tag.contains(thing)`, or all the objects in the tag queried with `tag.getAllElements()`.
 
-As an example:
+A reminder that `Tag`s contain `Item`s not `ItemStack`s. You can get the `Item` of an `ItemStack` from the stack by calling `<itemStack>.getItem()`.
+
+An example of checking if an Item belongs to a Vanilla `Tag`:
 ```Java
+//Retrieve whatever tag you want (using Tags.Blocks or Tags.Items)
+Tag<Item> tag = Tags.Items.DUSTS_REDSTONE;
+boolean inTag;
+// There's two methods to do this, doesn't really matter which you choose:
+// Method 1: from the Item's perspective
+inTag = someItemStack.getItem().isIn(tag);
+// Method 2: from the Tag's perspective
+inTag = tag.contains(someItemStack.getItem());
+```
+
+An example of checking if an Item is contained in a certain Forge `Tag`:
+```Java
+// Create a custom resource to use to fetch a tag instance ('mymod:myitemgroup' is the tag)
 ResourceLocation myTagId = new ResourceLocation("mymod", "myitemgroup");
+// Tags work through Items, not ItemStacks so we need to get the Item from our ItemStack
 Item unknownItem = stack.getItem();
-boolean isInGroup = ItemTags.getCollection().getOrCreateTag(myTagId).contains(unknownItem);
-// alternatively, can use getTag and perform a null check
+// This line will either retrieve the Tag (referenced by our ResourceLocation) or create it if it is not
+// in the ItemTags collection of all the tags. Then it will check if our Item (again, not ItemStack)
+// exists in the set of Items that belong to this Tag
+boolean isInGroup = ItemTags.getCollection().getOrCreate(myTagId).contains(unknownItem);
+// Alternatively, you can use the getTag and perform a null check to see if the tag exists and do it that way
```

Comment:
This isn't worth mentioning as it should never be done. There is no point, an empty tag and a non-existent tag are treated the same.

Reply:
I'm not sure what you mean, this is from the previously existing documentation but with extra comments?

Reply:
That doesn't make it correct. If the goal here is to *improve* this page, it should be correct.